### ISSUE TITLE
TypeError: Cannot read property 'toLowerCase' of null

### ISSUE THREAD

**Issue by jerome-benoit** [LucasBrazi06/ev-dashboard#2051]
```
ERROR TypeError: Cannot read property 'toLowerCase' of null
    at AppTransactionsFormatConnector.buildConnectorText (transactions-connector-cell.component.ts:93)
    at AppTransactionsFormatConnector.transform (transactions-connector-cell.component.ts:35)
    at pureFunction2Internal (core.js:36733)
    at Module.ɵɵpipeBind2 (core.js:36915)
    at TransactionsConnectorCellComponent_Template (transactions-connector-cell.component.ts:13)
    at executeTemplate (core.js:12169)
    at refreshView (core.js:12008)
    at refreshComponent (core.js:13458)
    at refreshChildComponents (core.js:11729)
    at refreshView (core.js:12064)
```

**Comment by LucasBrazi06** [comment]
Where did you get this error?

**Comment by LucasBrazi06** [comment]
Fixed in b0f84bcd73b9e28b340e7ea4c05f0f9706cb3c12

**Comment by LucasBrazi06** [comment]
Don't open an issue that will take you more time to create than to provide a fix ;-)
